Makefile

```diff
@@ -57,6 +57,7 @@ INSTALL_RPMS_FROM_URL ?=
 
 CHECK_COVERAGE ?=0
 RUN_TESTS ?= 0
+BUILD_TESTS ?= 0
```

Comment:
Set to 1 if RUN_TESTS == 1?

Reply:
@atobiszei BUILD_TESTS refer to the build stage together with ovms target. Typically the build image is done without running tests and tests are executed in a container. This param was added to speedup the CI pipeline and reuse remote build cache and to avoid building tests for if they are not needed. 
If someone would include RUN_TESTS=1 in the build phase. It will trigger tests execution script which is also building the tests if missing. But this is not recommended for performance reasons as it would require also to download all models and install all optimum...

Reply:
If this only steers when the tests are build I am more inclide to more concrete name like
BUILD_TESTS_TOGETHER?
Otherwise it introduces confustion IMO. @mzegla  thoughts?

Reply:
@dtrawins but configuration with BUILD_TESTS == 0 and RUN_TESTS == 1 is invalid right?

Reply:
@mzegla it is valid, RUN_TEST will force building the tests. It will be built while launching the tests instead of building the tests together with ovms target.

Reply:
Well, I got confused, so I guess it proves @atobiszei point. Such naming makes me think they are codependent.
Maybe we can be more verbose like:
`BUILD_TESTS` -> `BUILD_WITH_TESTS`
`RUN_TESTS` -> `BUILD_AND_RUN_TESTS`
Or something else that makes it a bit clearer